appium/ios/appium-ios-tests.js

```diff
@@ -0,0 +1,252 @@
+const wdio = require("webdriverio");
+const assert = require('chai').assert;
+
+const opts = {
+	path: '/wd/hub',
+	port: 4723,
+	connectionRetryTimeout: 900000,
+	wdaLaunchTimeout: 900000,
+	capabilities: {
+        platformName: "iOS",
+        platformVersion: "15.0",
+        deviceName: "iPhone 11",		
+        automationName: "XCUITest",
+		app: "/Users/runner/work/mobile-pipeline-poc/mobile-pipeline-poc/ReactNativeSemaphoreNew.xcarchive/Products/Applications/ReactNativeSemaphoreNew.app",
```

Comment:
Could we fix the indentation of these lines?

Reply:
It is properly formatted in the repo [link](https://github.com/liatrio/mobile-pipeline-poc/blob/appium-workflow-integration/appium/ios/appium-ios-tests.js) but not showing up in the pr.